benchmarks/GraphBLAS-sharp.Benchmarks/BenchmarksMxv.fs

```diff
@@ -3,16 +3,13 @@ namespace GraphBLAS.FSharp.Benchmarks
 open GraphBLAS.FSharp
 open GraphBLAS.FSharp.Backend
 open BenchmarkDotNet.Attributes
-open BenchmarkDotNet.Configs
-open BenchmarkDotNet.Columns
-open System.IO
-open GraphBLAS.FSharp.IO
+open GraphBLAS.FSharp.Backend.Objects
 
 [<Config(typeof<CommonConfig>)>]
 type MxvBenchmarks() =
     let rand = System.Random()
 
-    let mutable matrix = Unchecked.defaultof<Matrix<float>>
+    let mutable matrix = Unchecked.defaultof<ClMatrix<float>>
```

Comment:
`Cl` prefix looks like a bad idea. What is the reason to add it?

Reply:
For consistency with vector: https://github.com/YaccConstructor/GraphBLAS-sharp/blob/2addd2b2f4c8e94133c90c27dfb1ee15413f919f/src/GraphBLAS-sharp.Backend/Objects/Vector.fs#L94